### ISSUE TITLE
rename the preset

### ISSUE THREAD

**Issue by chnuessli** [dbrgn/josm-preset-potmch#1]
Nice work there. I think you can rename it and try to put a PR in the JOSM project for long lasting environment.

Wäre schade wäre das "nur" für PotM

**Comment by dbrgn** [comment]
Yes, I'll probably do that after the PotM finished, with a few tweaks (e.g. removing the CH-specific list of networks, and the SEV sockets). There already is a charging station preset, but that can be merged.

**Comment by chnuessli** [comment]
Is this done?

**Comment by dbrgn** [comment]
I've renamed the repository and the file.

Now my idea is to either make a generic preset, or to improve the one shipped with JOSM. Do you know where I can find the JOSM preset sources?

**Comment by chnuessli** [comment]
I just know this sources:
- https://github.com/simonpoole/beautified-JOSM-preset
- https://github.com/JOSM/josm/blob/master/resources/data/defaultpresets.xml

**Comment by chnuessli** [comment]
HI @dbrgn can i help you? Or is this issue still pending?

**Comment by dbrgn** [comment]
@chnuessli hmm, to be honest I'm not quite sure how to proceed. The preset can be used as-is and works great for Switzerland and probably other central european countries as well. However, the JOSM preset must be more general.

I think a few aspects can be updated in the JOSM preset as well. I'll send a PR soon. However, there's probably still some value in this separate preset.

**Comment by dbrgn** [comment]
Done: https://github.com/JOSM/josm/pull/127

**Comment by chnuessli** [comment]
Thank you
